Packs/ApiModules/Scripts/NGINXApiModule/NGINXApiModule.py

```diff
@@ -0,0 +1,305 @@
+
```

Comment:
@roysagi can you add it to EIS and EDL and make sure they work ok with the API module?

Reply:
I have added this to EIS but I think we should add it to EDL in a different PR in order to keep this PR slim for the bug fix.